### PR TITLE
Add snippet for sort.Sort implementation

### PR DESCRIPTION
**Pull request by lukebjerring**
See https://golang.org/pkg/sort/

Example output: $1: ByName, $2: Badge, $3: a[i].Name < a[j].Name

    type ByName []Badge

    func (a ByName) Len() int           { return len(a) }
    func (a ByName) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
    func (a ByName) Less(i, j int) bool { return a[i].Name < a[j].Name }

### CODE REVIEW

File: snippets/go.json [hunk 1]
```diff
@@ -250,5 +250,10 @@
 			"body": "package main\n\nimport (\n\t\"fmt\"\n\t\"net/http\"\n\t\"time\"\n)\n\nfunc greet(w http.ResponseWriter, r *http.Request) {\n\tfmt.Fprintf(w, \"Hello World! %s\", time.Now())\n}\n\nfunc main() {\n\thttp.HandleFunc(\"/\", greet)\n\thttp.ListenAndServe(\":8080\", nil)\n}",
 			"description": "Snippet for sample hello world webapp"
 		}
```
Comment (lloiser) [inline snippets/go.json]: The snippet file is invalid because there is a comma missing here.

Comment (lukebjerring) [reply snippets/go.json]:
```suggestion
		},
```

**Comment by jhendrixMSFT** [comment]
@vladbarosan anything to add?

**Comment by vladbarosan** [comment]
looks good

**Pull request merged**